src/test/java/core/basesyntax/service/implementations/CsvFileReaderTest.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax.service.implementations;
+
+import core.basesyntax.service.FileReader;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class CsvFileReaderTest {
+    public static final String EMPTY_FILE_PATH = "src/main/resources/FileReaderTest.csv";
+    public static FileReader reader;
+
+    @BeforeClass
+    public static void initialize() {
+        reader = new CsvFileReader();
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void emptyFile_ThrowsException() {
+        reader.getAllLines(EMPTY_FILE_PATH);
+    }
+
+    @Test(expected = RuntimeException.class)
+    public void wrongPath_ThrowsException() {
+        reader.getAllLines("FUUUUUUUUUUUUUU");
+    }
+
+    @Test
+    public void getAllLines_Correct() {
+        List<String> list = reader.getAllLines("src/main/resources/test1.CSV");
```

Comment:
```suggestion
        List<String> list = reader.getAllLines("src/test/resources/test1.csv");
```
resources that are used in tests should be located in `src/test` folder